src/wayland/protocols/corner_radius.rs

```diff
@@ -45,6 +45,11 @@ impl CornerRadiusState {
 }
 
 pub trait CornerRadiusHandler {
+    fn add_corners(
+        &mut self,
+        toplevel: &XdgToplevel,
+        toplevel_obj: cosmic_corner_radius_toplevel_v1::CosmicCornerRadiusToplevelV1,
+    );
     fn corner_radius_state(&mut self) -> &mut CornerRadiusState;
     fn toplevel_from_resource(&mut self, toplevel: &XdgToplevel) -> Option<ToplevelSurface>;
```

Comment:
I think we can also get rid of this function, if we just make the trait depend on the `XdgShellHandler`-trait from smithay?